### PR TITLE
Add icon for QQ

### PR DESCRIPTION
**Pull request by Fldicoahkiin**
## Description

https://play.google.com/store/apps/details?id=com.tencent.mobileqq

## Type of change

:x: Bug fix (non-breaking change which fixes an issue)
✅ Icon addition (non-breaking change that adds/modifies Lawnicons's icons)
:x: General change (non-breaking change that doesn't fit the above categories like copyediting)

### Icons added
* QQ (`com.tencent.mobileqq`) - added from icon request form

### Icons linked
* App Name (linked `com.tencent.mobileqq` to `@drawable/qq`)

### CODE REVIEW

**Review by daywalk3r666 — COMMENTED** [review]
Icon must be 192x192 and a content size of 160x160. The stroke must be 8 or 12

**Comment by Fldicoahkiin** [comment]
> Icon must be 192x192 and a content size of 160x160. The stroke must be 8 or 12

I have already rework the QQ icon according to the Icon guidelines.This is my first icon,please check again to see if it meets the specifications

**Comment by daywalk3r666** [comment]
For me looks now good but i'm not a maintainer. You must wait for a review from @Chefski

**Comment by Fldicoahkiin** [comment]
Also,I have a question: can I change the brush stroke to 10px? After my test, this size is the most appropriate for the effect on the phones.

File: app/src/main/res/xml/grayscale_icon_map.xml
Comment (Goooler) [inline app/src/main/res/xml/grayscale_icon_map.xml:978]:
```suggestion
    <icon drawable="@drawable/qq" package="com.tencent.mobileqq" name="QQ" />
```

**Review by Chefski — APPROVED** [review]

**Comment by Chefski** [comment]
Lawnicons only uses 12px and 8px (with occasional 6px for small parts)

> Also,I have a question: can I change the brush stroke to 10px? After my test, this size is the most appropriate for the effect on the phones.

**Pull request merged**
